packages/backend/src/workers/action.ts

```diff
@@ -61,6 +62,28 @@ export const worker = new Worker(
     prefix: '{actionQ}',
     connection: createRedisClient(),
     concurrency: appConfig.workerActionConcurrency,
+    settings: {
+      backoffStrategy: (
+        attemptsMade: number,
+        type: string,
+        err: Error,
```

Comment:
I was actually hoping to inspect errno (which Axis exposes via AxiosError, and we throw that from our response interceptor.) instead of string parsing, but for some reason that's all disappearred when it reaches us. Suspect bullMQ is re-reconstructing our error object somewhere :(

Reply:
Yes, I tried that too. So bullmq actually wraps the error so bopian